applications/GeoMechanicsApplication/custom_constitutive/small_strain_umat_2D_interface_law.cpp

```diff
@@ -127,4 +127,26 @@ void SmallStrainUMAT2DInterfaceLaw::SetValue(const Variable<Vector>& rVariable,
     }
 }
 
+SizeType SmallStrainUMAT2DInterfaceLaw::WorkingSpaceDimension() { return Dimension; }
+
+SizeType SmallStrainUMAT2DInterfaceLaw::GetStrainSize() const { return VoigtSize; }
+
+ConstitutiveLaw::StrainMeasure SmallStrainUMAT2DInterfaceLaw::GetStrainMeasure()
+{
+    return StrainMeasure_Infinitesimal;
+}
+
+ConstitutiveLaw::StressMeasure SmallStrainUMAT2DInterfaceLaw::GetStressMeasure()
+{
+    return StressMeasure_Cauchy;
+}
+
+std::string SmallStrainUMAT2DInterfaceLaw::Info() const { return "SmallStrainUMAT2DInterfaceLaw"; }
```

Comment:
We may want to use string literals here (which avoid implicit conversions from `const char*` to `std::string`):
```suggestion
std::string SmallStrainUMAT2DInterfaceLaw::Info() const { return "SmallStrainUMAT2DInterfaceLaw"s; }
```
To make this work, you need to add `using namespace std::string_literals;` near the top of the file.

Naturally, this suggestion also applies to the `Info` members of the other classes that have been moved from the header to the implementation file. Thank you.

Reply:
Thank you for the suggestion. It is applied in this PR and for processes.